nautobot/extras/tables.py

```diff
@@ -918,3 +919,34 @@ class Meta(BaseTable.Meta):
             "http_content_type",
             "enabled",
         )
+
+
+class JobHookTable(BaseTable):
+    pk = ToggleColumn()
+    name = tables.Column(linkify=True)
+    content_types = tables.TemplateColumn(WEBHOOK_CONTENT_TYPES)
+    job = tables.Column(linkify=True)
+    enabled = BooleanColumn()
+    type_create = BooleanColumn()
+    type_update = BooleanColumn()
+    type_delete = BooleanColumn()
```

Comment:
Do these not just function correctly as-is from the model? Seems to me the only ones that would need to be overloaded are `pk` and `content_type`?

Reply:
`name` and `job` didn't get linkify set. The others were fine to remove.